client/my-sites/people/followers-list/index.jsx

```diff
@@ -201,13 +192,23 @@ let Followers = React.createClass( {
 		} else {
 			followers = this.renderPlaceholders();
 		}
+
+		const downloadListLink = this.props.fetchOptions.type === 'email'
```

Comment:
Perhaps we should check to make sure `this.props.site` is defined before this line, just in case.  I hit some errors with a empty localStore when loading this page ( not related to the changes here ).  But regardless, this button shouldn't be rendered if there is not site.ID as the link will not be valid.